pogo_gui.py

```diff
@@ -0,0 +1,306 @@
+from tkinter import *
+from tkinter import ttk
+from tkinter import filedialog
+import psycopg2
+
+from multi_poke_v1 import *
+
+
+def db_search(search_value):
+    # search for pokemon names that match search string
+    # make results global so can be accessed in other functions
+    global search_db
+
+    # create connection to database
+    db = psycopg2.connect(database = 'mydb')
+    cur = db.cursor()
+
+    cur.execute(
+        "SELECT species FROM base_stats WHERE species ILIKE (%s)", (search_value+'%',))
+    search_db = cur.fetchall()
+    # returns a list of tuples: [('Lapras',), ('Lanturn',), ('Larvitar',)]
+    # use this sum method to convert into list: ['Lapras', 'Lanturn', 'Larvitar']
+    search_db = list(sum(search_db, ()))
+    search_db.sort()
+    db.close()
+
+def set_single_search_results():
+    # display database search results in listbox for single pokemon
+    try:
+        single_list_results.set(search_db) 
+    except Exception as e:
+        print(e)
+
+
+def set_evo_search_results():
+    # display database search results in listbox for evolution pokemon
+    try:
+        # set listbox 
+        search_results.set(search_db)
+        if len(search_db) > 10:
+            list_results.set(search_db[0:10])
+        else:
+            list_results.set(search_db)
+    except Exception as e:
+        print(e)
+
+def onKeyReleaseSingle(event):
+    # on key press, performs search of normal pokemon string in database
+    search_value = pokemon_name.get()
+    db_search(search_value)
+    set_single_search_results()
+
+def onKeyRelease(event):
+    # on key press, performs search of evolution pokemon string in database
+
+    search_value = search_chosen.get()
+    db_search(search_value)
+    set_evo_search_results()
+
+def onLeftClick(event):
+    # on left click, make list selection evolution pokemon
+
+    # curselection returns tuple of index of item chosen: (2,) or second list item
+    selection_tuple = result_list.curselection()
+
+    try:
+        # cast index as int after extracting from tuple
+        selection_idx = int(selection_tuple[0])
+
+        # get list of search results from combobox values (somehow only global list)
+        select_from = search_db
+        selection = select_from[selection_idx]
+
+        # set search box to list selection
+        search_chosen.set(selection)
+
+    except:
+        # nothing is selected, empty tuple
+        pass
```

Comment:
I'm not sure what might land in this exception, but it might be useful to printout some warning if the user needs to fix their inputs due to an exception.